plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/DeployJob.java

```diff
@@ -85,13 +80,11 @@
   private final boolean includeOptionalConfigurationFiles;
   private final CollectingLineListener errorCollectingLineListener;
 
-  public StandardDeployJob(IProject project,
-                           Credential credential,
-                           IPath workDirectory,
-                           ProcessOutputLineListener stagingStdoutLineListener,
-                           ProcessOutputLineListener stderrLineListener,
-                           DefaultDeployConfiguration deployConfiguration,
-                           boolean includeOptionalConfigurationFiles) {
+  public DeployJob(IProject project, Credential credential, IPath workDirectory,
+      ProcessOutputLineListener stagingStdoutLineListener,
```

Comment:
Why is this stagingStdoutLineListener but the next argument is only stderrLineListener? I suggest simply stdoutLineListener but if there's a reason these are different, Javadoc might help.